_posts/2023-06-21-#27748.md

```diff
@@ -10,16 +10,75 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- This PR is a follow-on to [PR 25325](https://github.com/bitcoin/bitcoin/pull/25325),
+  which we [reviewed](https://bitcoincore.reviews/25325) March 8 of this year.
+  Please review at least the Notes for that review club.
+
+- It's important that the coins cache uses the amount of memory indicated by
```

Comment:
`-dbcache` is [not just the coins cache](https://github.com/bitcoin/bitcoin/blob/1ecdf6ea8fb6dc1d64f41323a1e337beb174c25b/src/node/caches.cpp#L12-L34) so I think this needs a bit of rephrasing?